datasafe-storage/datasafe-storage-impl-fs/src/test/java/de/adorsys/datasafe/business/impl/storage/FileSystemStorageServiceTest.java

```diff
@@ -37,6 +46,75 @@ void prepare(@TempDir Path dir) {
         );
     }
 
+    @Test
+    void objectExists() {
+        createFileWithMessage();
+        assertThat(storageService.objectExists(storageService.list(root).findFirst().get()));
```

Comment:
Complete assertion:
assertThat(storageService.objectExists(storageService.list(root).findFirst().get())).isTrue();

Reply:
Hi Valentyn, 
it looks like we have this wong usage of assertThat quit a lot of time:

        assertThat(profileRetrievalService.userExists(userJohn)).isFalse();
        assertThat(profileRetrievalService.userExists(john.getUserID()));
        assertThat(profileRetrievalService.userExists(john.getUserID()));
        assertThat(result).isEqualTo(MESSAGE_ONE);
        assertThat(privateJane.getResource().asPrivate().decryptedPath()).asString().isEqualTo(PRIVATE_FILE_PATH);
        assertThat(privateJane.getResource().asPrivate().encryptedPath()).asString().isNotEqualTo(PRIVATE_FILE_PATH);
        assertThat(inbox).hasSize(1);
        assertThat(foundResource.location()).isEqualTo(expectedInboxResource.location());
        assertThat(foundResource.location().getPath()).asString().contains(SHARED_FILE);
        assertThat(storage.read(foundResource)).asString().doesNotContain(MESSAGE_ONE);
        assertThat(privateFiles).hasSize(1);
        assertThat(foundResource.location()).isEqualTo(expectedPrivateResource.location());
        assertThat(foundResource.toString()).doesNotContain(PRIVATE_FILE);
        assertThat(foundResource.toString()).doesNotContain(FOLDER);
        assertThat(storage.read(foundResource)).asString().doesNotContain(MESSAGE_ONE);
            assertThat(resourceList.size()).isEqualTo(EXPECTED_NUMBER_OF_FILES_PER_USER);
        assertThat(privateJohnFiles).hasSize(2);
            assertThat(testPath).contains(item.getResource().asPrivate().decryptedPath().getPath());
            assertThat(expectedData.contains(data)).isTrue();
        assertThat(readingResult).isEqualTo(MESSAGE_THREE);
        assertThat(directResult).isEqualTo(MESSAGE_THREE);
        assertThat(latest.getResource().asPrivate().decryptedPath()).asString().contains(PRIVATE_FILE_PATH);
        assertThat(listRoot(jane)).isEmpty();
        assertThat(versionedResource).hasSize(3);
        assertThat(versionedResource)
        assertThat(readingResult).isEqualTo(MESSAGE_THREE);
        assertThat(TEST_MESSAGE_CONTENT).isEqualTo(new String(actualResult));
        assertThat(decryptionStream).hasContent(MESSAGE_CONTENT);
        assertThat(s.contains("null")).isTrue();
        assertThat(inbox.list(request)).hasSize(1);
        assertThat(inbox.read(request)).hasContent(BYTES);
        assertThat(captor.getValue().getLocation()).isEqualTo(resource);
        assertThat(outputStream.toByteArray()).contains(BYTES.getBytes());
        assertThat(resource.location()).isEqualTo(absolute.location());
        assertThat(resource.location()).asString().isEqualTo("s3://root/" + ENCRYPTED);
        assertThat(captor.getValue().encryptedPath()).asString().isEqualTo(ENCRYPTED);
        assertThat(captor.getValue().decryptedPath()).asString().isEqualTo("path");
        assertThat(resource.location()).asString().isEqualTo("s3://root/bucket/" + ENCRYPTED);
        assertThat(resource.getResource().decryptedPath()).asString().isEqualTo(DECRYPTED);
        assertThat(resource.getResource().encryptedPath()).asString().isEqualTo(ENCRYPTED);
        assertThat(resource.location()).asString().isEqualTo("s3://root/bucket/path/" + ENCRYPTED);
        assertThat(resource.getResource().decryptedPath()).asString().isEqualTo(DECRYPTED);
        assertThat(resource.getResource().encryptedPath()).asString().isEqualTo("path/" + ENCRYPTED);
        assertThat(privateService.list(request)).hasSize(1);
        assertThat(inbox.read(request)).hasContent(BYTES);
        assertThat(captor.getValue().getLocation()).isEqualTo(resource);
        assertThat(outputStream.toByteArray()).contains(BYTES.getBytes());
        assertThat(storageService.objectExists(storageService.list(root).findFirst().get())).isTrue();
        assertThat(storageService.list(nonExistingFileLocation).collect(Collectors.toList()).isEmpty()).isTrue();
        assertThat(storageService.objectExists(newFileLocation));
        assertThat(storageService.list(root).collect(Collectors.toList()).isEmpty());
        assertThat(!storageService.list(root).collect(Collectors.toList()).isEmpty());
        assertThat(storageService.list(root))
        assertThat(storageService.list(root)).isEmpty();
        assertThat(storageService.read(fileWithMsg)).hasContent(MESSAGE);
        assertThat(storageService.read(fileWithMsg)).hasContent(MESSAGE);
        assertThat(Paths.get(fileWithMsg.location())).exists();
        assertThat(Paths.get(fileWithMsg.location())).doesNotExist();
        assertThat(storageService.list(root))
        assertThat(storageService.list(root)).isEmpty();
        assertThat(storageService.read(fileWithMsg)).hasContent(MESSAGE);
        assertThat(storageService.read(fileWithMsg)).hasContent(MESSAGE);
        assertThat(Log.secure(TEST_STRING)).isEqualTo(TEST_STRING);
        assertThat(Log.secure(TEST_STRING)).isEqualTo("/p****le");
        assertThat(out).isNotEqualTo(TEST_STRING);
        assertThat(out.length()).isEqualTo(44);
        assertThat(Log.secure(Paths.get(TEST_STRING))).isEqualTo("fi****e:///****/****/****");
        assertThat(Log.secure(TEST_URI)).isEqualTo("ht****p://ww****om/un****rm/re****ce/id****er");
        assertThat(Log.secure(path) == null);
        assertThat(Log.secure(uri) == null);
        assertThat(Log.secure(uri) == null);
        assertThat(Log.secure(resource)).isEqualTo("ht****p://ww****om/un****rm/re****ce/id****er");
        assertThat(Log.secure(TEST_URI_ENDS_SLASH)).isEqualTo("ht****p://ww****om/un****rm/re****ce/id****er/");
        assertThat(Log.secure(resource)).isEqualTo("ht****p://ww****om/un****rm/re****ce/id****er/");

I am going to fix this too. The files are
./datasafe-business/src/test/java/de/adorsys/datasafe/business/impl/e2e/BasicFunctionalityTest.java
./datasafe-business/src/test/java/de/adorsys/datasafe/business/impl/e2e/BasicFunctionalityWithConcurrencyTest.java
./datasafe-business/src/test/java/de/adorsys/datasafe/business/impl/e2e/VersionedDataTest.java
./datasafe-encryption/datasafe-encryption-impl/src/test/java/de/adorsys/datasafe/business/impl/cmsencryption/CmsEncryptionServiceImplTest.java
./datasafe-encryption/datasafe-encryption-impl/src/test/java/de/adorsys/datasafe/business/impl/cmsencryption/SymetricEncryptionTest.java
./datasafe-encryption/datasafe-encryption-impl/src/test/java/de/adorsys/datasafe/business/impl/keystore/KeyStoreAuthTest.java
./datasafe-inbox/datasafe-inbox-impl/src/test/java/de/adorsys/datasafe/business/api/inbox/actions/ListInboxImplTest.java
./datasafe-inbox/datasafe-inbox-impl/src/test/java/de/adorsys/datasafe/business/api/inbox/actions/ReadFromInboxImplTest.java
./datasafe-inbox/datasafe-inbox-impl/src/test/java/de/adorsys/datasafe/business/api/inbox/actions/WriteToInboxImplTest.java
./datasafe-privatestore/datasafe-privatestore-impl/src/test/java/de/adorsys/datasafe/business/impl/privatespace/actions/EncryptedResourceResolverTest.java
./datasafe-privatestore/datasafe-privatestore-impl/src/test/java/de/adorsys/datasafe/business/impl/privatespace/actions/ListPrivateImplTest.java
./datasafe-privatestore/datasafe-privatestore-impl/src/test/java/de/adorsys/datasafe/business/impl/privatespace/actions/ReadFromPrivateImplTest.java
./datasafe-privatestore/datasafe-privatestore-impl/src/test/java/de/adorsys/datasafe/business/impl/privatespace/actions/WriteToPrivateImplTest.java
./datasafe-storage/datasafe-storage-impl-fs/src/test/java/de/adorsys/datasafe/business/impl/storage/FileSystemStorageServiceTest.java
./datasafe-storage/datasafe-storage-impl-s3/src/test/java/de/adorsys/datasafe/business/impl/storage/S3SystemStorageServiceTest.java
./datasafe-types/src/test/java/de/adorsys/datasafe/business/api/types/utils/LogTest.java